integrations/visual-website-optimizer/lib/index.js

```diff
@@ -24,7 +24,8 @@ var VWO = (module.exports = integration('Visual Website Optimizer')
   .option('useExistingJQuery', false)
   .option('replay', true)
   .option('listen', false)
-  .option('experimentNonInteraction', false));
+  .option('experimentNonInteraction', false)
+  .option('isSPA', false));
```

Comment:
I like the idea of letting the customers choose if they want to send events for all experiments. I'm aware this wasn't on the initial requirements, but it should be easy to add: Let's add a new setting `trackOnlyABExperiments` with default value `false`, as:
```javascript
.option('trackOnlyABExperiments'));
```